lib/bindings/MQTTBinding.js

```diff
@@ -211,6 +212,22 @@ function singleMeasure(apiKey, deviceId, attribute, device, message) {
     });
 }
 
+function updateCommand(apiKey, deviceId, device, message) {
```

Comment:
Same as before... Is it really needed to pass the `deviceId` to this function as argument since already included in the `device` one?